controller/telemetry/server.go

```diff
@@ -249,6 +214,27 @@ func (s *server) ListPods(ctx context.Context, req *read.ListPodsRequest) (*publ
 		return nil, err
 	}
 
+	// Reports is map from instance name to the absolute time of the most recent
+	// report from that instance.
+	reports := make(map[string]time.Time)
+	// Query Prometheus for reports in the last 30 seconds.
+	res, err := s.prometheusAPI.Query(ctx, "reports_total[30s]", time.Time{})
```

Comment:
we were previously doing `expiry := time.Now().Add(-10 * time.Minute)` ... i believe `reports_total[30s]` means we're expiring after 30s?

Reply:
that's right.  Given that reports are supposed to come at least every 10s, a 30s TTL seemed more reasonable to me than 10m.  WDYT?